canban-react/src/components/Column.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import "../style.css";
+import Card from './Card.js';
+import PropTypes from 'prop-types';
+
+export default class Column extends React.Component{
+	addCards(){
```

Comment:
this is the wrong name for a function with such content. If you are having "add" prefix, it mains that this is an action, but currently it's not. You can use "render" prefix for example

Reply:
Also, this code can be moved to the new component